.github/workflows/ci.yml

```diff
@@ -23,8 +23,101 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test:
-    name: "Run pipeline with test data (${{ matrix.NXF_VER }} | ${{ matrix.test_name }} | ${{ matrix.profile }})"
+  nf-test-gpu:
+    name: "Run GPU Tests (${{ matrix.NXF_VER }} | ${{ matrix.profile }})"
+    # Only run on push if this is the nf-core dev branch (merged PRs)
+    if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/methylseq') }}"
+    runs-on: "gpu"
+    strategy:
+      fail-fast: false
+      matrix:
+        NXF_VER:
+          - "24.04.2"
+        shard: [1, 2]
+        filter: [pipeline, workflow]
+        profile:
+          - "docker"
+          - "singularity"
+        isMaster:
+          - ${{ github.base_ref == 'master' }}
+        # Exclude singularity on dev
+        exclude:
+          - isMaster: false
+            profile: "singularity"
+    env:
+      NXF_ANSI_LOG: false
```

Comment:
no need for this one, it's already defined L13

Reply:
Also, we're running in nf-test, so we probably don't need ansi-log set up to false at all